decidim-elections/db/migrate/20201026163334_add_blocked_at_and_bb_status_to_elections.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddBlockedAtAndBbStatusToElections < ActiveRecord::Migration[5.2]
+  def change
+    add_column :decidim_elections_elections, :blocked_at, :datetime
+    add_column :decidim_elections_elections, :bb_status, :string
```

Comment:
@tramuntanal, We are using a hash as argument for enum, and this stores strings in the database so the hash can be changed in the future with no big issues, also having this allows us to use all the Rails enum helper methods.

The default value is `nil`, as the value comes from the bulletin board that has not reached yet. Both have the same enum, and they should always be in sync. 

https://github.com/decidim/decidim/blob/336ddb4f024c249d698192b956ee13b24e7a2a54/decidim-elections/app/models/decidim/elections/election.rb#L19


https://github.com/decidim/decidim-bulletin-board/blob/a9e311c03494b5d0859c525fa760f7f306d40125/decidim-bulletin-board-app/app/models/election.rb#L9

Reply:
ah, it seems this is not well documented: https://api.rubyonrails.org/classes/ActiveRecord/Enum.html
but you're right, its much better to have it persisted as a string because probably the performance improvement to use an integer can't be that much